src/main/comments.js

```diff
@@ -1001,11 +1012,13 @@ function printTrailingComment(commentPath, print, options) {
     return lineSuffix(
       concat([hardline, isLineBeforeEmpty ? hardline : "", contents])
     );
-  } else if (isBlock) {
+  } else if (isBlock || isParentSuperClass) {
     // Trailing block comments never need a newline
+    // console.log("trailing block comment");
     return concat([" ", contents]);
   }
 
+  // console.log("trailing comment");
```

Comment:
Could you delete these comments

Reply:
oh man that's embarassing...
Fixed now :)